tests/common/fixtures/populate_fdb.py

```diff
@@ -0,0 +1,120 @@
+import json
+import logging
+import pytest
+
+from common.platform.ssh_utils import prepare_testbed_ssh_keys as prepareTestbedSshKeys
+from ptf_runner import ptf_runner
+
+logger = logging.getLogger(__name__)
+
+class PopulateFdb:
+    """
+        PopulateFdb populates DUT FDB entries
+
+        It accepts MAC to IP ratio (default 100:1) and packet count (default 2000). It generates packets with
```

Comment:
It would be helpful to specifically call out the command line arguments that are being used by this class.

Reply:
Thanks, done